dev-db/sqlitebrowser/sqlitebrowser-3.8.0-r1.ebuild

```diff
@@ -21,7 +21,7 @@ RDEPEND="
 	dev-cpp/antlr-cpp:2
 	dev-db/sqlite:3
 	dev-libs/qcustomplot[qt5=]
-	x11-libs/qscintilla
+	x11-libs/qscintilla:=
```

Comment:
It appears that ``dev-db/sqlitebrowser`` bundles qscintilla. If it didn't it would crash immediately anyway due to symbol collisions between qt4 and qt5.